src/dsql/parse.y

```diff
@@ -863,6 +863,7 @@ using namespace Firebird;
 	Jrd::SetDecFloatTrapsNode* setDecFloatTrapsNode;
 	Jrd::SetBindNode* setBindNode;
 	Jrd::SessionResetNode* sessionResetNode;
+	Jrd::RecreatePackageBodyNode* recreatePackageBodyNode;
```

Comment:
Can this declaration be eliminated if you declare `replace_package_body_clause` to return a `ddlNode`?

Reply:
Good idea